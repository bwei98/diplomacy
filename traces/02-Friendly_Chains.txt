# Spring 1901
# England and France tests are basic; Russia is a longer chain; Turkey tests cycle
F Edi - NTH; A Lvp - Edi
F Bre - MAO; A Par - Bre

A War - Mos; A Mos - Sev; F Sev - BLA


F Ank - Con; A Con - Smy; A Smy - Ank
#
# END