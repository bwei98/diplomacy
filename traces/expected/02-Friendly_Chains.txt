# Spring 1901
F NTH; A Lvp; F Lon
Lvp; Edi; Lon
F MAO; A Bre; A Mar
Par; Bre; Mar
F Kie; A Ber; A Mun
Kie; Ber; Mun
F Stp; A Mos; A Sev; F BLA
Mos; Stp; Sev; War
F Tri; A Vie; A Bud
Vie; Tri; Bud
A Ven; A Rom; F Nap
Rom; Ven; Nap
A Smy; A Ank; F Con
Smy; Ank; Con
#
# END